CONTRIBUTING.md

```diff
@@ -4,22 +4,38 @@
 
 If you would like to contribute enhancements or fixes, please do the following:
 
-1.  Fork the package repository.
-2.  Run `npm install` to setup all dependencies.
-3.  Hack on a separate topic branch created from the latest `master`.
-4.  Check for lint errors with `npm run lint` or use `linter-eslint` within Atom.
-5.  Run the package specs.
-6.  Commit the changes and push the topic branch to your fork.
-7.  Make a pull request to the main repo.
-8.  Welcome to the club!
+1. Fork the package repository and clone your fork to local file.
+
+1. Run `npm install` in the project to setup all dependencies.
+
+1. Run `apm link --dev` to install your working folder as a development-mode-only package in Atom.
```

Comment:
"install" -> "link"
(`install` implies it's a separate copy)

Reply:
Hmmm. Yeah I got a bit hung up on that sentence. I was trying to get across that it would have the same effect as an install. 😕

Reply:
Initially I was trying to keep it pretty short to match the brevity of all the other steps. But after further editing, some of the other steps got longer too, which means some more text would not be out of place anymore. So how about this?

> Run `apm link --dev` to link your working folder as a development-mode-only package. Atom will load the package as if you installed it to `{user_home_directory}.atom\dev\packages\`.

Reply:
The second part should be something like "Atom will load this development version instead of any regularly installed package, allowing you to test your code in a live environment."

(You can't install a package in `~/.atom/dev/packages`, that's _only_ for testing development packages, including overriding any version that is currently installed in `~/.atom/packages`.

Reply:
Works for me.